app/components/LegoReactions/index.tsx

```diff
@@ -58,32 +63,51 @@ const LegoReactions = (props: Props) => {
     });
   }
 
+  let usersByReaction = {};
+
+  if (parentEntity.reactionsGrouped && parentEntity.reactions) {
+    for (let groupedReaction of parentEntity.reactionsGrouped) {
+      for (let reaction of parentEntity.reactions) {
```

Comment:
The nested loop here isn't the most optimal, but I don't know if it is worth to refactor.

If there are many reactions and many grouped reactions, this could lead to a quadratic time complexity O(n^2).

Some potential improvements could be:
* Create a map or object to keep track of which emojis have already been processed, thus avoiding the need for a nested loop.
* Iterate over parentEntity.reactions just once, and add the author to the respective emoji array.